espresso/src/main/java/com/elpassion/android/commons/espresso/matchers/EditTextErrorMatcher.kt

```diff
@@ -0,0 +1,23 @@
+package com.elpassion.android.commons.espresso.matchers
+
+import android.support.annotation.StringRes
+import android.support.design.widget.TextInputLayout
+import android.support.test.espresso.matcher.BoundedMatcher
+import android.view.View
+import android.widget.EditText
+import org.hamcrest.Description
+
+class EditTextErrorMatcher(@StringRes private val textId: Int) : BoundedMatcher<View, EditText>(EditText::class.java) {
```

Comment:
Change `textId ` to `errorrStringId` for consistency or change TextInputEditTextHintMatcher field to `textId`.